src/explore-education-statistics-common/src/components/form/FormSelect.tsx

```diff
@@ -27,8 +27,9 @@ export interface FormSelectProps
   error?: string;
   id: string;
   inline?: boolean;
+  inlineHint?: boolean;
```

Comment:
Is this prop necessary? Would have assume that this can be inferred from `inline` and `hint` being set?

Reply:
It is, they do different things:
- `inline` makes the label and select be inline, in this case the hint is rendered below them (it's usually between the label and select)
`inlineHint` makes the hint be inline with the label, it can only be used when then label and select aren't inline.

I've added comments to Props to explain this.